db/db_tunables.h

```diff
@@ -1236,6 +1236,10 @@ REGISTER_TUNABLE("debug.omit_blob_write",
                  "Deliberately corrupt insertion randomly to debug db_verify", TUNABLE_BOOLEAN,
                  &gbl_debug_omit_blob_write, INTERNAL, NULL, NULL, NULL,
                  NULL);
+REGISTER_TUNABLE("debug.allow_constraint_violation",
+                 "Deliberately allow insertion without constraint chec to debug db_verify", TUNABLE_BOOLEAN,
+                 &gbl_debug_allow_constraint_violation, INTERNAL, NULL, NULL, NULL,
```

Comment:
typo here?  'chec' should be 'check' ?

Reply:
yep

Reply:
I was just thinking that maybe the name of this tunable should more closely reflect what it actually does... Perhaps something like "debug.skip_constraints_on_insert"...  I don't know.

Reply:
sure...will do